frontend/app/configuration/Config.scala

```diff
@@ -152,6 +152,9 @@ object Config {
   val googleAnalyticsTrackingId = config.getString("google.analytics.tracking.id")
 
   val youtubeMembershipVerificationId = config.getString("youtube.membership.verification.id")
+  val youtubeUrl = config.getString("youtube.url")
+
+  val googleplusUrl = config.getString("googleplus.url")
```

Comment:
These don't need to be configuration! Maybe have a `Social`  object which just has these as constant val fields.